consensus/src/round_manager.rs

```diff
@@ -989,12 +997,28 @@ impl RoundManager {
         }
     }
 
-    pub async fn process_verified_proposal(&mut self, proposal: Block) -> anyhow::Result<()> {
-        let proposal_round = proposal.round();
+    async fn create_vote(&mut self, proposal: Block) -> anyhow::Result<Vote> {
         let vote = self
             .vote_block(proposal)
             .await
             .context("[RoundManager] Process proposal")?;
+
+        fail_point!("consensus::create_vote", |_| {
```

Comment:
create_vote -> create_invalid_vote

Reply:
The function is actually used to a create a correct vote for production. It just has a fail_point that returns a bad vote for the sake of smoke tests.

Reply:
i mean for this string "consensus::create_vote" failpoint

Reply:
Done.